<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharProperty</class>
 <widget class="QDialog" name="CharProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>配置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QCheckBox" name="trainPathCheckBox">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>设为默认值</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="trainPathButton">
          <property name="text">
           <string>…</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="trainPathEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="trainPathLabel">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>训练库路径：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QPushButton" name="resultPathButton">
          <property name="text">
           <string>…</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="resultPatLabel">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>成果保存路径：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="resultPathEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="2">
         <widget class="QLineEdit" name="resultNameEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="resultNameLabel">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>默认成果文件名：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QCheckBox" name="resultLoadCheckBox">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>设为读取时的默认值</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="resultPathCheckBox">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>设为保存时的默认值</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="testPathLabel">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>测试集路径：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="testPathEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="testPathButton">
          <property name="text">
           <string>…</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="testPathCheckBox">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>设为默认值</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="addedPercentage">
        <item>
         <widget class="QCheckBox" name="generateTestSetCheckBox">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>从训练集中抽取测试集 抽取比例：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="testSetPercentage"/>
        </item>
        <item>
         <widget class="QLabel" name="percentLabel">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="allowedMultipleLabel">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>样本集差异倍数最大值：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="plateMultiplePercentage">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="plateMultiplePercentLabel">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="minPlateSampleLabel">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>最小样本数（均衡化时少于该数量的样本集会被无视）：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="minPlateSampleBox">
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="singleStep">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="similarytyCheck">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>导入训练库时剔除相似图像</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="6" column="2">
         <widget class="QPushButton" name="OKButton">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="letterSimilarity">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>字母相似度阈值：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="thresholdForNonChar">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="layout">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="numSimilarity">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>数字相似度阈值：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QDoubleSpinBox" name="thresholdForLetter">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="noncharSimilarity">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>非字符相似度阈值：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="thresholdForNum">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QDoubleSpinBox" name="thresholdForCnchar">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="cncharSimilarity">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>数字相似度阈值：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
